data/mouseSpecificRxns.tsv

```diff
@@ -9,3 +9,4 @@ rxns	equations	ub	lb	eccodes	subSystems	grRules	rxnReferences	rxnConfidenceScore
 "MA_20008"	"O2[r] + L-gulono-1,4-lactone[r] => H2O2[r] + ascorbate[r] + H+[r]"	1000	0	"1.1.3.8"	"Ascorbate and aldarate metabolism"	"Gulo"	"PMID:14962674"	4	"R10053"	"RXN-13689"	""	"MNXR116004"
 "MA_20009"	"O2[r] + L-gulono-1,4-lactone[r] => H2O2[r] + L-xylo-Hexulonolactone[r]"	1000	0	"1.1.3.8"	"Ascorbate and aldarate metabolism"	"Gulo"	""	2	"R03184"	"L-GULONOLACTONE-OXIDASE-RXN"	""	"MNXR140097"
 "MA_20010"	"PAPS[c] + aniline[c] => PAP[c] + Benzenamine sulfate[c] + H+[c]"	1000	0	"2.8.2.3"	"Amine sulfotransferase"	"Sult3a1"	""	2	"R01861"	""	""	"MNXR107208"
+"MA_20016"	"ascorbate[r] => ascorbate[c]"	1000	0	""	"Transport"	""	""	0	""	""	""	""

```

Comment:
I'm surprised to see this transport reaction as being species-specific. However, it doesn't have any other external identifiers, and no literature reference. Perhaps then it is the result of gap-filling? And if so, shouldn't this reaction be part of Human-GEM - ie, is it really species specific?

Reply:
yes, this addition can be viewed as a result of gap-filling for achieving the metabolic task #257 (Biosynthesis of Vitamin C) defined in [metabolicTasks_Full.xlsx](https://github.com/SysBioChalmers/Human-GEM/tree/master/data/metabolicTasks). And it's not part of Human-GEM, since it's species-specific.

Reply:
Seems like this reaction should be reversible, but I realize that's something that should probably be addressed in Human-GEM where it originates.

Reply:
This transportation from ER to cytosol is added for modelling purpose. If there is any reference reporting this transporter and indicating the reversibility? @JonathanRob

Reply:
Ah, sorry, I didn't realize that the reaction didn't come from Human-GEM. I don't know of any specific references, but [this paper](https://pubmed.ncbi.nlm.nih.gov/24907663/) is probably a good place to start. For now though I think it's fine to keep the gap-filled reaction as you have it.